profile.md

```diff
@@ -1459,13 +1459,13 @@ In this example, the `nonce`, `preferred_username`, `name`, and `email` claims a
      RAF still refers to it as eduPersonAssurance, but it will probably change into `eduperson_assurance`, following the OIDCre whitepaper.
 
 [^10]:
-     [https://openid.net/specs/openid-connect-federation-1_0.html](https://openid.net/specs/openid-connect-federation-1_0.html)
+     [OIDC federation](https://openid.net/specs/openid-connect-federation-1_0.html)
```

Comment:
OpenID-Connect Federation has been renamed into OpenID Federation, i.e. without the Connect, and the link should be adapted to be https://openid.net/specs/openid-federation-1_0.html